content/en/server/linux-desktop.md

```diff
@@ -8,50 +8,17 @@ id = "desktop_on_linux"
 
 +++
 
-## Desktop on Linux {.page-first-header}
+## Running Shoko Desktop on Linux with Wine using Bottles {.page-first-header}
 
-1. Download the latest Shoko Desktop latest (zip) from Github.
-    - Extracting to ```~/``` works fine, but you can also extract directly into the Wine environtment's virtual C:/ later.
+1. Install Bottles (https://usebottles.com). This may require installing Flatpak and adding the Flathub repository if you don't have it already.
+2. Before adding a bottle, go to Bottles preferences and under the "Runners" tab, install the latest Caffe or Proton GE. Other runners may also be installed, but I have verified that Caffe and Proton GE work. Other runners might work and might give better performance. I just know that the Soda runner can be slow and certain errors might popup that prevent Shoko Desktop being utilized for all its functions.
+3. Add a Bottle and give it a name like "Shoko Desktop" and under environment, select "Custom" and then choose one of the runners that were installed in the last step (Caffe or Proton GE). Then hit "Create." It may take a second for the Bottle to be done creating.
+4. Select the Bottle and under Options > Dependencies, install dotnetcoredesktop6, cjkfonts, and tahoma32.
+5. Next under Options > Settings > Components, enable DXVK and VKD3D. This is to hopefully increase compatibility, but not might be strictly necessary. It might help with reducing the application from going black which can be fixed by resizing the Shoko Desktop window when it's running.
+6. Lastly, download Shoko Desktop, extract the zip, and put it somewhere you remember. Then in Bottles under the current configuration, select Programs > Add Shortcuts... and navigate to where you downloaded Shoko Desktop and select the ShokoDesktop.exe.
```

Comment:
The zip is only for dailies iirc. There are only installers for stable releases. Feel free to correct me if I'm wrong.

Reply:
Currently checking if Bottles plays fine with the installer, since stable Desktop doesn't have a portable zip unlike the dailies.